test/stdlib/subString.swift

```diff
@@ -29,7 +29,6 @@ SubstringTests.test("Equality") {
   expectEqual("fg" as String, s.suffix(2))
   
 #if _runtime(_ObjC)
```

Comment:
I'm not sure why all these substring tests are guarded under `_runtime(_ObjC)`. Is that a mistake, @milseman?

Reply:
(in which case availability would be a better tool for this)